.release-please-manifest.json

```diff
@@ -4,5 +4,6 @@
   "packages/google-cloud-beyondcorp-appgateways": "0.1.0",
   "packages/google-cloud-beyondcorp-appconnections": "0.1.0",
   "packages/google-cloud-beyondcorp-clientconnectorservices": "0.1.0",
-  "packages/google-cloud-beyondcorp-clientgateways": "0.1.0"
+  "packages/google-cloud-beyondcorp-clientgateways": "0.1.0",
+  "packages/typeless-sample-bot": "0.1.0"
```

Comment:
I would leave these at 0.0.0 or the initial release won't bump the version in package.json (which our monorepo publish tool uses to identify which components to release)